features/provider/admin/messages/outbox.feature

```diff
@@ -1,24 +0,0 @@
-Feature: Outbox messages
```

Comment:
Why is the whole file removed?

Reply:
These test are covered here test/functional/developer_portal/admin/messages/outbox_controller_test.rb which makes more sense

Reply:
But the tests are slightly different. I wouldn't remove the tests that already exist and validate functionality.

Reply:
Actually, this test seems to be for the **provider** outbox page, rather than the **developer portal**. And this test breaks with the change.